### DISCUSSION TITLE
Custom Scripts Issue - Theme switching perpetually dark mode

### DISCUSSION THREAD

**Discussion by CodingPupper3033** [AutoDarkMode/Windows-Auto-Night-Mode#586] [Custom scripts]
Hello,

I have created a custom script to change my mouse cursor to match the theme (white on light, black on dark). It worked for a while, however somewhat recently stopped working. I tested the script to ensure it wasn't an issue with windows; however, it still works. My mouse is perpetually stuck in dark mode. When switching from dark mode to light mode, the cursor changes for about half a second until it reverts. It appears that, for some bizarre reason, AutoDarkMode is executing a dark mode command right after a light mode command. Does anyone know why that could be? I can send my files, logs, scripts, and everything if you would like.

Thanks!

**Comment by CodingPupper3033** [comment]
I believe it may now be sending the command with no parameters after it changes to light mode for some reason, very strange, but I will check to see if this is the issue

**Reply by CodingPupper3033** [reply]
I am looking further into it, and it seems like I may have an issue with my script, but at this point, I don't think I will fix it, just not worth the time

**Reply by Spiritreader** [reply]
To me it looks like you have Theme mode enabled and you have this option set
![image](https://user-images.githubusercontent.com/18438899/199721770-3c197b4c-cc95-4c82-8f86-127e7051bacb.png)

Changing your cursor changes the active theme, causing ADM to switch back to what was set.
Try unchecking that in case it was enabled.

If it's not that then I'd probably point towards something in the script as well.

Also if you want to know what your script outputs and what precisely ADM does to invoke your script, calling arguments and output are logged in the adm config file.
